public/docs/_examples/cb-a11y/ts/app/a11y-index.component.html

```diff
@@ -0,0 +1,7 @@
+<nav role="navigation">
+    <ol>
+        <li>
+            <a href="" [routerLink]="['FormControls']"><h2>Accessible form control labels</h2></a>
```

Comment:
This was purely my own OCD. Works 100% fine without adding it. Started doing this expecially for code metrics on the code base with checkers that do not understand Angular.